quarkchain/cluster/miner.py

```diff
@@ -289,11 +291,18 @@ def _mine_new_block_async(self):
         block = copy.deepcopy(self.work_map[header_hash])
         header = block.header
         header.nonce, header.mixhash = nonce, mixhash
-
         # sign as a guardian
         if self.guardian_private_key and isinstance(block, RootBlock):
             header.sign_with_private_key(self.guardian_private_key)
 
+        # remote sign as a guardian
+        if isinstance(block, RootBlock) and header.signature == bytes(65) and signature != bytes(65):
```

Comment:
1. use [precommit + black](https://github.com/QuarkChain/pyquarkchain#development-flow) for code formatting
2. how about following? seems we don't need guardian pub key

```python
if isinstance(block, RootBlock):
  header.signature = signature
```